test/unit/__snapshots__/finances.test.ts.snap

```diff
@@ -1650,7 +1650,7 @@ Array [
                 "CurrencyAmount": 3.38,
                 "CurrencyCode": "USD",
               },
-              "ProductDescription": "Shacke Metal Shoe Horn 7.5&quot; inches - Double Sided Stainless
+              "ProductDescription": "Shacke Metal Shoe Horn 7.5\\" inches - Double Sided Stainless
```

Comment:
That's weird. I think both parsers are wrong here. Original one retained HTML symbols. The new one just replaces it with \\ 

The only explanation I can think of is that maybe HTML escaped symbols aren't valid in xml?

Reply:
I think this is correct

`&quot;` = `"` 

so it's backslash to negate the following backslash to negate the `"`

Reply:
So to me it looks like it's replacing the HTML entities to their `string` counterpart